### PR TITLE
Add jwtAccess to report

### PR DESCRIPTION
**Pull request by kolanos**
The `jwtAccess` is a signed JWT that contains the bucket and object key for the profiling report.

### CODE REVIEW

**Comment by kolanos** [comment]
Removing this from v1.2.0 until further review.

File: iopipe/contrib/profiler/plugin.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 class ProfilerPlugin(Plugin):
     name = 'profiler'
-    version = '0.1.0'
-    homepage = 'https://github.com/iopipe/iopipe-python'
+    version = '0.2.1'
```
Comment (pselle) [inline iopipe/contrib/profiler/plugin.py]: Shouldn't this go to 0.2.0 instead of 0.2.1?

File: iopipe/contrib/profiler/plugin.py [hunk 2]
```diff
@@ -53,11 +54,13 @@ def post_invoke(self, event, context):
...
-        pass
-
-    def post_report(self, report):
         if self.profile is not None:
             with tempfile.NamedTemporaryFile() as stats_file:
                 self.profile.dump_stats(stats_file.name)
                 signed_request = get_signed_request(report)
                 upload_profiler_report(signed_request['signedRequest'], stats_file.file)
+                if 'jwtAccess' in signed_request:
+                    self.context.iopipe.log('@iopipe/profiler.jwtAccess', signed_request['jwtAccess'])
```
Comment (pselle) [inline iopipe/contrib/profiler/plugin.py]: This is saving the value in custom metrics? Don't we want to add this to the schema?

**Comment by kolanos** [comment]
@pselle This has been updated to use `plugins[].uploads[]`.

**Comment by kolanos** [comment]
@pselle I'm going to hold off on merging this until we have jwtAccess support in gql. Going to update the extension in this PR, which would make the gql update a blocker.

**Pull request merged**